src/patternfly/components/FormControl/form-control.scss

```
@@ -67,12 +41,12 @@ $pf-v5-c-form-control--m-clock--Coordinates: "M256 8C119 8 8 119 8 256s111 248 2
   --#{$form-control}--placeholder--child--Color: var(--#{$pf-global}--Color--100);
 
   // input disabled style
-  --#{$form-control}--disabled--Color: var(--#{$pf-global}--disabled-color--100);
-  --#{$form-control}--disabled--BackgroundColor: var(--#{$pf-global}--disabled-color--300);
-  --#{$form-control}--disabled--BorderColor: transparent;
+  --#{$form-control}--m-disabled--Color: var(--#{$pf-global}--m-disabled-color--100);
```

The global vars need the `m-` removed, since that will break the styling.

```suggestion
  --#{$form-control}--m-disabled--Color: var(--#{$pf-global}--disabled-color--100);
```